lnmuxrpc/lnmux.proto

```diff
@@ -94,7 +94,7 @@ message Invoice {
     bytes preimage = 2;
     uint64 amount_msat = 3;
     bool settled = 4;
-    uint64 settled_requested_at = 5;
+    uint64 settle_requested_at = 5;
```

Comment:
Does the deposit connector need to be updated too?

Reply:
No, this `message` is only use for the ListInvoices rpc call